tools/bazelwrapper

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+# Make sure we're in the project root directory.
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+pushd "$SCRIPTPATH/.." > /dev/null
+trap popd > /dev/null ERR EXIT
+
+BAZEL_VERSION="0.12.0rc1"
+BAZEL_VERSION_SHA="fe9bf6e4d920c668f28a37d5d2ea11cee38871e90ccf5c2a52c7a327dd6de9a9"
+BAZEL_VERSION_URL="https://releases.bazel.build/0.12.0/rc1/bazel-0.12.0rc1-installer-darwin-x86_64.sh"
```

Comment:
Want to point to the latest release? 0.13.0

Reply:
I've got a handful of tools building with and using the same version of Bazel - it's nice to atomically update them all at once. Perhaps this script can become it's own repo 👍